src/current/_includes/v23.2/ui/refresh.md

```diff
@@ -0,0 +1,13 @@
+### Refresh
+
+{% include_cached new-in.html version="v23.2" %} To control refresh of the data on the Active Executions views of the SQL Activity Statements and Transactions pages, the following controls have been added
```

Comment:
Copy editing comments:

- Add a colon after "following controls have been added"
- Capitalize all the sentences in the bulleted list (starting with "an","a","an")

Reply:
updated